src/material.cpp

```diff
@@ -1436,4 +1491,72 @@ openmc_extend_materials(int32_t n, int32_t* index_start, int32_t* index_end)
 
 extern "C" size_t n_materials() { return model::materials.size(); }
 
+//
+// cvmt functions 
+double 
+PolyProperty::evaluate(Position r) {
+  double results {0.0}; 
+  if(type_ == "zernike1d"){ 
+    results = evaluate_zernike1d(r);
+  } else if (type_ == "zernike"){
+    results = evaluate_zernike(r);
+   }
+  return results;
```

Comment:
You can just use `return` directly above instead of setting a temporary variable.